modules/administration-guide/pages/trusted-extensions-for-microsoft-visual-studio-code.adoc

```diff
@@ -0,0 +1,67 @@
+:_content-type: PROCEDURE
+:description: Configure trusted extensions for Microsoft Visual Studio Code
+:keywords: extensions, vs-code, vsx, open-vsx, marketplace
+:navtitle: Configure trusted extensions for Microsoft Visual Studio Code
+
+[id="visual-studio-code-trusted-extensions"]
+= Configure trusted extensions for Microsoft Visual Studio Code
+
+
+You can use the `trustedExtensionAuthAccess` field in the `product.json` file of Visual Studio Code to specify which extensions are trusted to access authentication tokens.
+[source,json]
+----
+	"trustedExtensionAuthAccess": [
+		"<publisher1>.<extension1>",
+		"<publisher2>.<extension2>"
+	]
+----
+
+This is particularly useful when you have extensions that require access to services such as GitHub, Microsoft, or any other service that requires OAuth. By adding the extension IDs to this field, you are granting them the permission to access these tokens.
+
+[WARNING]
+====
+Use the `trustedExtensionAuthAccess` field with caution as it could potentially lead to security risks if misused. Give access only to trusted extensions.
+====
+
+.Procedure
+[IMPORTANT]
+====
+Since the Visual Studio Code editor is bundled within `che-code` image, you can only change the `product.json` file when the workspace is started up.
```

Comment:
```suggestion
Since the Microsoft Visual Studio Code editor is bundled within `che-code` image, you can only change the `product.json` file when the workspace is started up.
```